tests/TestCase.php

```diff
@@ -101,6 +101,8 @@ protected function createDummySnapshots()
             $this->disk->put("snapshot{$i}.sql", $this->getSnapshotContent("snapshot{$i}"));
         }
 
+        sleep(1);
```

Comment:
Make sure that snapshot4 has a timestamp dat is at least 1 second later than the others, so it will sort correctly.